tutorials/btp-cf-buildpacks-python-create/btp-cf-buildpacks-python-create.md

```diff
@@ -5,12 +5,13 @@ title: Create a Python Application via Cloud Foundry Command Line Interface
 description: Create a simple Python application in the Cloud Foundry Command Line Interface (cf CLI) and enable services for it.
 auto_validation: true
 time: 40
-tags: [ tutorial>beginner, software-product>sap-btp--cloud-foundry-environment, software-product>sap-hana]
+tags: [ tutorial>beginner, software-product>sap-btp--cloud-foundry-environment, software-product>sap-hana, software-product-function>sap-btp-cockpit]
 primary_tag: programming-tool>python
 ---
 
 ## Prerequisites
```

Comment:
Please remove the full stops from the following lines.

Reply:
Done!